src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/DefaultTexts/index.js

```diff
@@ -1,40 +1,64 @@
 /* eslint-disable react/no-array-index-key */
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Link, Heading } from '@datapunt/asc-ui';
 
 import { defaultTextsType } from 'shared/types';
 
-import './style.scss';
+const StyledH4 = styled(Heading)`
+  font-weight: normal;
+  margin-top: 20px;
+  margin-bottom: 8px;
```

Comment:
use themeSpacing(2) here

Reply:
every Heading has a bottom margin that needs to be put in  the component inside ASC in:
they should be:
H1: 30px
H2: 12px
H3: 12px
H4: 8px
 When they are applied in ASC we will remove the extra bottom margin here.

Reply:
gedaan